README.md

```diff
@@ -14,18 +14,55 @@ Implement the tool `st` that does exact pattern matching using a suffix tree. Te
 # Report
 
 ## Specify if you have used a linear time or quadratic time algorithm.
+We ended up with a quadratic time algorithm, we played around with mccreight, but other projects got the better of us, and ate what little time we had left.
```

Comment:
You really should give algorithms top priority, but I know you are immune to good advice.